### PR TITLE
feat: add more examples

### PR DESCRIPTION
**Pull request by MulL1on**
What type of PR is this?

feat: add more examples

What this PR does / why we need it (English/Chinese):

en: add detailed examples according to the CloudWeGo documentation, adding a POST request example
ch: 根据 CloudWeGo 官方文档添加了更详细的例子，添加了一个 POST 请求的例子

Which issue(s) this PR fixes:

https://github.com/cloudwego/hertz/issues/485

### CODE REVIEW

File: eureka/example/custom_conn_mutiple_server/client/client.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+		status, body, err := cli.Get(context.Background(), nil, "http://hertz.discovery.eureka/ping", config.WithSD(true), config.WithTag("key1", "val1"))
+		if err != nil {
+			hlog.Fatal(err)
+		}
+		hlog.Infof("code=%d,body=%s", status, string(body))
+	}
+}
+
+func discoveryWithCustomizedAddr(r discovery.Resolver) {
+	fmt.Println("discovery with customizedAddr:")
```
Comment (li-jin-gou) [inline eureka/example/custom_conn_mutiple_server/client/client.go:89]: 统一到 hlog

File: eureka/example/custom_conn_mutiple_server/client/client.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+	cli, err := client.NewClient()
+	if err != nil {
+		panic(err)
+	}
+
+	cli.Use(sd.Discovery(r, sd.WithCustomizedAddrs(net.JoinHostPort("127.0.0.1", "5001"))))
+	for i := 0; i < 10; i++ {
+		status, body, err := cli.Get(context.Background(), nil, "http://hertz.discovery.eureka/ping", config.WithSD(true), config.WithTag("key1", "val1"))
```
Comment (li-jin-gou) [inline eureka/example/custom_conn_mutiple_server/client/client.go:97]: 统一到 Do

Comment (li-jin-gou) [reply eureka/example/custom_conn_mutiple_server/client/client.go:97]: 原因是实际观察到用户的反馈 GET 用的很少。

**Comment by li-jin-gou** [comment]
consul need to fix https://github.com/hashicorp/consul/issues/17973

File: eureka/example/custom_conn_mutiple_server/client/client.go (hunk 1 above)
Comment (MulL1on) [reply eureka/example/custom_conn_mutiple_server/client/client.go]: done

File: eureka/example/custom_conn_mutiple_server/client/client.go (hunk 2 above)
Comment (MulL1on) [reply eureka/example/custom_conn_mutiple_server/client/client.go]: done

**Review by li-jin-gou — APPROVED** [review]
LGTM

**Pull request merged**